shared/src/helpers/date/subtract-years/subtract-years.helper.ts

```diff
@@ -0,0 +1,7 @@
+import { subYears } from 'date-fns';
+
+const subtractYears = (amount: number, date?: Date): Date => {
+  return date ? subYears(date, amount) : subYears(new Date(), amount);
```

Comment:
```suggestion
const subtractYears = (amount: number, date: Date): Date => {
  return subYears(new Date(), amount);
```
you need to check it before the call of the function